src/c/address-form.js

```diff
@@ -10,13 +10,14 @@ const I18nScope = _.partial(h.i18nScope, 'activerecord.attributes.address');
 
 const addressForm = {
     controller(args) {
+        const parsedErrors = args.parsedErrors;
         const countriesLoader = postgrest.loader(models.country.getPageOptions()),
             statesLoader = postgrest.loader(models.state.getPageOptions()),
             countries = m.prop(),
-            defaultCountryID = 36,
+            defaultCountryID = 36, // @TODO get id from endpoint
```

Comment:
this looks more like a configuration entry. we also have something similar [on the payment-vm](https://github.com/catarse/catarse.js/blob/427657b039f6d9556d8d899c02dff795f4231dfe/src/vms/payment-vm.js#L55).